### PR TITLE
Checkbox tests

### PR DESCRIPTION
**Pull request by KRRISH96**
1) Docs Updated.(Issue #30 fixed)
2) Render, Accessibility and edge case tests added..

### CODE REVIEW

File: src/checkbox/index.js [hunk 1]
```diff
@@ -20,7 +20,9 @@ const Checkbox = ({
     <label className={classNames}>
       <input
         type="checkbox"
+        name={label} // when name prop isn't passed, name sets to same value as label prop
```
Comment (prasanna1211) [inline src/checkbox/index.js:23]: This doesn't allow customized name right ?

File: src/checkbox/index.js [hunk 2]
```diff
@@ -20,7 +20,9 @@ const Checkbox = ({
...
         onClick={() => onClick(label, label)}
+        id={checked ? 'checked' : undefined}
```
Comment (prasanna1211) [inline src/checkbox/index.js:25]: Why do you need this conditional statement ?

File: src/checkbox/readMe.md [hunk 3]
```diff
@@ -1,63 +1,51 @@
...
-| `children` | `Any` | `null` | All children components to be rendered inside the button |
-| `type` | `String` | `default` | Represents the style to be applied for the button. Possible values are `primary`, `secondary`, `success`, `danger`, `warning`, `info`, `light`, and `dark` |
-| `size` | `String` | `medium`  | Defines the size of the button. Possible values are `small`, `medium`, and `large` |
-| `className` | `String` | `''` | Set a class to style the Component |
-| `href` | `String` | `null` | Creates a link for the button. |
-| `icon` | `String`/`Element` | `null` | Value of the icon |
-| `iconAlignment` | `String` | `left` | Set an alignment of the icon. Possible values are `left`, `right` |
-| `onClick` | `Function` | &nbsp; | Callback to be called when the button is clicked |
+| `checked` | `boolean` | `false` | Prop that sends in the state of toggle |
+| `onClick` | `Function` | &nbsp; | Callback to be called when checked |
```
Comment (prasanna1211) [inline src/checkbox/readMe.md:9]: "when checked or unchecked"

**Review by prasanna1211 — APPROVED** [review]

File: src/checkbox/index.js (hunk 1 above)
Comment (KRRISH96) [reply src/checkbox/index.js:23]: The user can pass in the `name`  prop, which overwrites the default `name={label}`

File: src/checkbox/index.js (hunk 2 above)
Comment (KRRISH96) [reply src/checkbox/index.js:25]: Fixed in #52

File: src/checkbox/readMe.md (hunk 3 above)
Comment (KRRISH96) [reply src/checkbox/readMe.md:9]: fixed in #52
